contracts/casino/include/casino/casino.hpp

```diff
@@ -145,6 +184,48 @@ class [[eosio::contract("casino")]] casino: public eosio::contract {
 
     void verify_game(uint64_t game_id);
     uint64_t get_game_id(name game_account);
+
+    void session_update(uint64_t ses_id, uint64_t game_id, asset quantity) {
+        session_state_table sessions(_self, _self.value);
+        const auto itr = sessions.find(ses_id);
```

Comment:
ses_id uniqueness guarantied only for particular game, so here we can potentially get ses_id collision

Reply:
so here I suppose approach without using ses_id:
- game contract updates max_win only by game_id
- then when session ends game contract will send max_win update with sum of whole max_wins for that session